journalbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,185 @@
+path. For example:
++
+["source","sh",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- paths:
+  - "/dev/log"
+  - "/var/log/messages/my-journal-file.journal"
+----
++
+If no paths are specified, {beatname_uc} reads from the default journal.
+
+. Set the <<{beatname_lc}-seek,`seek`>> option to control the position where
+{beatname_uc} starts reading the journal. The available options are `head`,
+`tail`, and `cursor`. The default is `cursor`, which means that
+{beatname_uc} will continue reading where it left off after a reload or restart.
+
+. (Optional) Set the <<{beatname_lc}-include-matches,`include_matches`>> option
+to filter entries in journald before collecting any log events. This reduces the
+number of fields that {beatname_uc} needs to process. For example, to fetch only
+Redis events from a Docker container tagged as `redis`, use: 
++
+["source","sh",subs="attributes"]
+----
+{beatname_lc}.inputs:
+- paths: []
+  include_matches:
+    - "CONTAINER_TAG=redis"
+    - "_COMM=redis"
+----
+
+include::../../libbeat/docs/step-configure-output.asciidoc[]
+
+include::../../libbeat/docs/step-configure-kibana-endpoint.asciidoc[]
+
+include::../../libbeat/docs/step-configure-credentials.asciidoc[]
+
+include::../../libbeat/docs/step-test-config.asciidoc[]
+
+include::../../libbeat/docs/step-look-at-config.asciidoc[]
+
+
+[id="{beatname_lc}-template"]
+=== Step 3: Load the index template in Elasticsearch
+
+include::../../libbeat/docs/shared-template-load.asciidoc[]
+
+[id="{beatname_lc}-starting"]
+=== Step 4: Start {beatname_uc}
+
+Start {beatname_uc} by issuing the appropriate command for your platform. If you
+are accessing a secured Elasticsearch cluster, make sure you've configured
+credentials as described in <<{beatname_lc}-configuration>>.
+
+NOTE: If you use an init.d script to start {beatname_uc} on deb or rpm, you can't
+specify command line flags (see <<command-line-options>>). To specify flags,
+start {beatname_uc} in the foreground.
+
+*deb and rpm:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+sudo service {beatname_lc} start
+----------------------------------------------------------------------
+
+*linux:*
+
+["source","sh",subs="attributes"]
+----------------------------------------------------------------------
+sudo chown root {beatname_lc}.yml <1>
+sudo ./{beatname_lc} -e -d "publish"
+----------------------------------------------------------------------
+<1> You'll be running {beatname_uc} as root, so you need to change ownership
+of the configuration file, or run {beatname_uc} with `--strict.perms=false`
+specified. See
+{libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
+in the _Beats Platform Reference_.
+
+// REVIEWERS: Do you think it's better to show the run command with or without
+// -c journalbeat.yml included? I'm inclined to show it without. WDYT?
```

Comment:
Checking docs the other beats are shown with `-c`. Personally I'm fine not using -c here.

Reply:
@urso I'm also removing the debug selector to match the changes that Andrew just made. (He thinks the debug output is too verbose, and I sort of agree, but I generally ignore output unless stuff doesn't run, so I'm not a good person to judge here.) Let me know if you would prefer that I keep -d.

Reply:
@dedemorton     I'm fine with removing `-d`